### ISSUE TITLE
panos_security_rule - update operation

### ISSUE THREAD

**Issue by LoganRoss747** [PaloAltoNetworks/ansible-pan#236]
I have several security rules (497) that need to be updated with a new log setting.  I created a lab to test why the attached playbook keeps failing in production.  I also attached the output with -vvv verbosity.  I get the same results in my lab firewall with one rule named "Test Spaces" as I do with the production firewall.

Same results happen with any option using the update operation.

[output.txt](https://github.com/PaloAltoNetworks/ansible-pan/files/2922613/error.2.txt)
[update-sec-rules.yml](https://github.com/PaloAltoNetworks/ansible-pan/files/2922607/update-sec-rules.txt)

**Comment by trustinger** [comment]
This happens to me as well. I know that `operation` is supposed to be deprecated at some point so Im guessing a dependency has been broken somewhere else. As a work around, Ive been using the core module `panos_set`:

```
    - panos_set:
        ip_address: "{{ inventory_hostname }}"
        username: "{{ username }}"
        password: "{{ password }}"
        command: set
        xpath: "/config/devices/entry[@name='localhost.localdomain']/vsys/entry[@name='vsys4']/rulebase/security/rules/entry[@name='{{ item['@name'] }}']"
        element: "<log-setting>test</log-setting>"
      with_items: "{{ pan_format.response.result.rules.entry }}"
```

**Comment by joshuawillis** [comment]
I did this to work around it... 
https://github.com/PaloAltoNetworks/ansible-pan/pull/265

**Comment by LoganRoss747** [comment]
This worked perfectly.

Thanks @joshuawillis

**Comment by shinmog** [comment]
This is now fixed by #265 so, re-closing this issue.
